src/test/java/com/google/cloud/tools/gradle/appengine/core/CheckCloudSdkTaskTest.java

```diff
@@ -143,6 +155,8 @@ public void testCheckCloudSdkAction_appEngineInstallationExceptions() {
     } catch (TaskExecutionException ex) {
       Assert.assertEquals(
           ex.getCause().getClass(), AppEngineJavaComponentsNotInstalledException.class);
+    } catch (CloudSdkNotFoundException e) {
```

Comment:
Should add this to the method signature instead of catching; I think we want the test to fail in this case since we're testing for a different exception. I think this one in particular is already tested in `testCheckCloudSdkAction_sdkInstallationException`.